config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presets.yaml

```diff
@@ -14,6 +14,7 @@ presets:
         key: serviceAccountSigningKey
 - labels:
     preset-azure-cred-wi: "true"
+  serviceAccountName: prowjob-default-sa
```

Comment:
it looks like we can't use presets for this level of configuration :-(

Reply:
so you mean we have to add it directly per-job config?

that's what I see on other jobs fwiw

Reply:
Checking if 
```yaml
spec:
	serviceAccountName: xyz
```
makes any difference.

Reply:
Spec will not be parsed. Ref: https://docs.prow.k8s.io/docs/jobs/#presets

Reply:
Adding it to per config. Will experiment if adding it to the job config via variable works.